Intersect (Core)/Config/Options.cs

```diff
@@ -29,11 +29,9 @@ public class Options
         [JsonProperty("OpenPortChecker", Order = 0)]
         protected bool _portChecker = true;
 
-        [JsonProperty("MaxClientConnections")]
-        protected int _maxConnections = 100;
+        [JsonProperty("MaxClientConnections")] protected int _maxConnections = 100;
 
-        [JsonProperty("MaximumLoggedinUsers")]
-        protected int _maxUsers = 50;
+        [JsonProperty("MaximumLoggedinUsers")] protected int _maxUsers = 50;
```

Comment:
Is there a reason you've changes these two lines like this?
Nothing around them is touched and I don't see a reason for this change related to the pr.

Reply:
My bad, i enabled some a feature that "fixes proper spacing and blank lines" at some point here, will undo all these asap.